app/scripts/controllers/containersgrid.js

```diff
@@ -93,6 +106,7 @@ angular.module('dockstore.ui')
       };
 
       $scope.$watch('searchQueryContainer', function(term) {
+        //console.log(term);
```

Comment:
Do not check in new commented code unless it is explained and has a high chance of being activated at some point.